api/src/zero/module/xx.clj

```diff
@@ -180,18 +201,26 @@
            (take max)
            (run! release!)))))
 
+(defn print-help
+  [& _]
+  (println description)
+  (println (describe-overrides "baits and targets" "-B" default-baits-and-targets)))
```

Comment:
- This being a function means that `describe-overrides` can help automate documenting this for users (I think)
- This accepting any arguments means we can slot it into the `(apply (case ...` below to simply print help text if the user supplies only a single argument (I think that's useful since I think this already needs to be a function)